coreos-4/policies/SA-System_and_Services_Acquisition/component.yaml

```diff
@@ -357,50 +357,94 @@
   narrative:
     - key: a
       text: |
-        A control response to SA-5(a) is planned. Progress
-        can be tracked at:
+        Obtaining admin docs are an organization policy and procedure control,
+        and are not applicable to OpenShift.
+        To assist with compliance efforts on behalf of organizations, documentation is
+        referenced below.
+        Documentation progress can be tracked at:
 
         https://issues.redhat.com/browse/CMP-367
+
     - key: a.1
       text: |
-        A control response to SA-5(a)(1) is planned. Progress
-        can be tracked at:
+        CoreOS installation documents:
+        https://coreos.com/docs/#tectonicInstallationHeader
+        CoreOS Hardening guide:
+        https://coreos.com/os/docs/latest/hardening-guide.html
+        Machine Config Operator documentation:
+        https://github.com/openshift/machine-config-operator/tree/master/docs
```

Comment:
url should be https://docs.openshift.com/container-platform/latest/nodes/nodes/nodes-nodes-managing.html